src/ratingservice/products.json

```diff
@@ -0,0 +1,110 @@
+{
+    "products": [
+        {
+            "id": "OLJCESPC7Z",
+            "name": "Vintage Typewriter",
+            "description": "This typewriter looks good in your living room.",
+            "picture": "/static/img/products/typewriter.jpg",
+            "priceUsd": {
+                "currencyCode": "USD",
+                "units": 67,
+                "nanos": 990000000
+            },
+            "categories": ["vintage"]
```

Comment:
Why do we need all of these fields in the product object that is only used by rating service?
Perhaps, id and name is enough for working with ratings? Do I miss anything?

Reply:
It's from the productcatalogservice. I've removed it.